dice.js

```diff
@@ -0,0 +1,41 @@
+// I'm like 90% sure this works ¯\_(ツ)_/¯
+
+const
+	_description = "Rolls dice, default 1 six-sided die - by cz4r",
+	_help = {
+		"/roll": "Roll the dice! '/roll' will roll a six-sided die, '/roll 5d8' will roll 5 eight-sided dice. - by cz4r"
+	}
+;
+
+var DicePlugin = function(data) {
+	var that = this;
+	that.bot = data.bot;
+	that.help = _help;
+	var str;
+	that.commands = {
+		"/roll": function(input) {
+			var total;
+			if(input) {
+				var vals = input.split("d");
+				if (parseInt(vals[0], 10) && parseInt(vals[1], 10)) {
+					for (i = 0; i < parseInt(vals[0]); i++ {
```

Comment:
Whoops, forgot the ) in the for loop. 

If you check out the updated file, we can remove the loop and just do the multiplication–saves us some cycles.